interfaces/openfga/v0/interface_tests/provider_tests.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Canonical
+# See LICENSE file for licensing details.
+import yaml
+from interface_tester.interface_test import SchemaConfig, interface_test_case
+from scenario import Relation, State
+
+
+@interface_test_case(
+    event='openfga-relation-created',
+    role='provider',
+    schema=SchemaConfig.empty
+)
+def test_no_data_on_created(output_state: State):
+    # nothing happens on created: databags are empty
+    return
+
+
+@interface_test_case(
+    event='openfga-relation-joined',
+    role='provider',
+    schema=SchemaConfig.empty
+)
+def test_no_data_on_joined(output_state: State):
+    # nothing happens on joined: databags are empty
+    return
+
+
+@interface_test_case(
+    event='openfga-relation-changed',
+    role='provider',
```

Comment:
I'm missing an analogous test case for the requirer

Reply:
added